aetcd/client.py

```diff
@@ -203,6 +271,20 @@ async def __aenter__(self):
     async def __aexit__(self, *args):
         await self.close()
 
+    @_handle_errors
+    @_ensure_connected
+    async def reset_auth_token(self) -> None:
```

Comment:
This probably should be refactored into private `_auth` and public `auth` and reused in `connect`.